guide/voice/seeking.md

```diff
@@ -0,0 +1,79 @@
+# Seek
+
+Currently, there's no way to seek audio resources with raw discord.js such as:
+
+```js
+const { createAudioResource } = require('@discordjs/voice');
+
+const resource = createAudioResource();
+
+resource.seek(1000); // This will be an error!
+```
+
+But some packages provide seek options with these solutions.
+
+## Cheat sheet
+
+Methods
+
+1. Play audio
+2. Create seeked resource
+3. Play the seeked audio resource
+
+Yo, this should be easy, right?
```

Comment:
this section is more like a TOC and lesser like a cheat sheet, could probably be replaced instead with the name of packages that are being used below?

Reply:
umm could u explain a bit more? like prism-media? or distube or discord-player?

Reply:
hm maybe this section can be completely removed as we do mention prism-media at the first section

Reply:
I got it